addons/binding/org.openhab.binding.yamahareceiver/docs/ReleaseNotes.md

```diff
@@ -0,0 +1,17 @@
+## Release Notes for Yamaha Receiver binding
+
+### 2.3.0
+
+* Fix [#2758](https://github.com/openhab/openhab2-addons/issues/2758): [YamahaReceiver] Problem with HDMI input.
+* Fix [#3034](https://github.com/openhab/openhab2-addons/issues/3034): [YamahaReceiver] VolumeDB is not updated.
+* Fix [#3062](https://github.com/openhab/openhab2-addons/issues/3062): [YamahaReceiver] ‘Zone_2’ of Yamaha HTR-xxx receiver completely incontrollable.
+* Fix [RX-V3900 compatibility](https://community.openhab.org/t/yamahareceiver-zone-discovery-not-working-with-rx-v3900/38350).
+* Fix Tuner: The Tuner play info was not working (tested on HTR-4069).
+* Feature: Added `Straight` surround program.
+* Feature: Introduced zone channel settings for minimum and maximum volume in dB.
+* Feature: Added setting for default album image URL.
+* Minor improvement (compatibility discovery, optimizations, [developer documentation](README.md)).
+* **Compatibility break**: Changed underlying settings name to follow the lower camel case convention (see discussion [here](https://github.com/openhab/openhab2-addons/pull/3264)).
```

Comment:
@davidgraeff  @martinvw here is the breaking change mentioned in the new release notes page. The binding documentation is also upgraded.

Reply:
Imho we should not have a releasenotes per binding, please add the breaking change to:

https://github.com/openhab/openhab-distro/wiki/%5BDRAFT%5D-openHAB-2.3-Release-Notes

The rest will were needed be included it in the global releasenotes but please do not add this file here.

Reply:
@zarusz if this is done I'm ready to merge :-)

Reply:
@martinvw lst file update [submitted](https://github.com/openhab/openhab-distro/pull/693), [global release wiki](https://github.com/openhab/openhab-distro/wiki/%5BDRAFT%5D-openHAB-2.3-Release-Notes) updated, and removed the release notes file from here (along with references).